### ISSUE TITLE
Dory cannot configure resolv.conf

### ISSUE THREAD

**Issue by Maxvanhattum** [FreedomBen/dory#59]
On ubuntu 20.04 and popos 21.10 dory cannot change/resolve the /etc/resolv/conf. See below.

If you edit this file manually and remove the comments that are present in the file it does work.

```Requesting sudo to write to /etc/resolv.conf
bash: line 20: nameserver: command not found
bash: line 21: nameserver: command not found
bash: line 22: options: command not found
bash: -c: line 23: unexpected EOF while looking for matching `''
bash: -c: line 24: syntax error: unexpected end of file
Error configuring local resolver  ```

**Comment by FreedomBen** [comment]
Finally figured out what this is.  On some distros there is a `'` in the resolv.conf file that doesn't get properly escaped.

Found on Fedora 36 in `/run/systemd/resolve/stub-resolv.conf`

Working on a fix but making a note here so this info doesn't get lost if I'm not able to finish it before getting pulled away for a higher priority (blasted day job)

**Comment by FreedomBen** [comment]
Fixed in ff631f206d18a9cb10c01d85c34e59fc6f7254a8

Released in v1.1.2:  5a43108b6a09bc6dc7b6907e692714ce3236805a

**Comment by FreedomBen** [comment]
@Maxvanhattum based on the error message I would imagine this fixes it for Ubuntu 20.04, but I tested on 22.04 so let me know if it's still an issue.  If it is, if you could share the output from:

```bash
ls --color --human-readable --classify --all -l /etc/resolv.conf
```
and
```bash
cat /etc/resolv.conf
```
that would be helpful
